javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,8 @@
+// Age-ify (A future age calculator)
+
+const YEAR_OF_BIRTH = 2002;
```

Comment:
we should always use camelCase for naming our variables, so it would be `yearOfBirth`